express/src/controllers/register.ts

```diff
@@ -19,6 +19,7 @@ export const processGetEmailForm: RequestHandler = async (req, res) => {
     console.info("In ProcessGetEmailForm");
 
     req.session.emailAddress = emailAddress;
+    req.session.save();
```

Comment:
Do we make assumptions anywhere as a result of the email being set? For example, if you entered the email if an already registered user here, would you be able to see some page you shouldn't?

Reply:
I expect the answer is no, but just want to make sure

Reply:
Good question, I think we can rule this out given that the user is still not authenticated and therefore no access token has been put into their session at this point.